data/test/scenarios/wml_tests/UnitsWML/AbilitiesWML/resistance/resistance_affect_everybody.cfg

```diff
@@ -0,0 +1,41 @@
+# wmllint: no translatables
+
+#####
+# API(s) being tested: [resistance]affect_allies=
+##
+# Actions:
+# Give all units 50% resistance to all damage types
```

Comment:
While it technically does match the test, "Give alice an ability which gives self and all adjacent units (thus everybody) 50% resistance to all damage types" would be clearer.

Reply:
Updated.